### ISSUE TITLE
what is your training epoch setting?

### ISSUE THREAD

**Issue by RuiboFan** [zzxslp/WCL#4]

**Comment by zzxslp** [comment]
If you are training with cross entropy + contrastive loss on top of a pretrained captioning model,  10-15 epochs should be enough. If you are training from scratch, we use 30 epochs as the default max epoch.
